esmvaltool/diag_scripts/seaborn_diag.py

```diff
@@ -443,6 +449,11 @@ def _set_legend_title(plot_obj, legend_title: str) -> None:
         legend = plot_obj.get_legend()
     elif hasattr(plot_obj, 'legend'):  # FacetGrid, PairGrid
         legend = plot_obj.legend
+    elif isinstance(plot_obj, sns.axisgrid.JointGrid):  # JointGrid workaround
```

Comment:
```suggestion
    elif isinstance(plot_obj, sns.axisgrid.JointGrid):  # JointGrid
```

If `JointGrid` also need a special treatment I wouldn't call this "workaround" 😉